internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -41,32 +41,14 @@ Parameters:
     Type: String
   TargetPort:
     Type: Number
-{{- if .NLB }}
-  NLBAliases:
```

Comment:
https://github.com/aws/copilot-cli/pull/3189 The reason why we added these parameters is to render the uri. Are the URI still showing up correctly after you removed the params?